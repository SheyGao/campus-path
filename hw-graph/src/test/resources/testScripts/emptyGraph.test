# Test that a graph can be created.

# Create a Graph
CreateGraph emptyGraph

# Print the nodes in the graph
ListNodes emptyGraph

