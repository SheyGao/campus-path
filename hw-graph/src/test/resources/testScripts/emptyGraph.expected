# Test that a graph can be created.

# Create a Graph
created graph emptyGraph

# Print the nodes in the graph
emptyGraph contains:
